ocs_ci/deployment/deployment.py

```diff
@@ -12,11 +12,12 @@
 
 from ocs_ci.deployment.ocp import OCPDeployment as BaseOCPDeployment
 from ocs_ci.framework import config
-from ocs_ci.ocs import constants, ocp, defaults, registry
+from ocs_ci.ocs import constants, ocp, defaults, registry, openshift_ops
```

Comment:
you shouldn't use openshift_ops. If the intention is to use the OCP class, use ocp.py's OCP class